release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -207,6 +207,13 @@ For more information about assigning IP addresses from an IP address pool to ser
 [id="ocp-4-13-osdk"]
 === Operator development
 
+[id="ocp-4-13-suggested-namespace-template"]
+==== Setting a suggested namespace template with default node selector
+
+With this release, Operator authors can set a default node selector on the suggested namespace where the Operator runs. The suggested namespace is created using the namespace manifest in YAML which is included in the `ClusterServiceVersion` (CSV).  When adding the Operator to a cluster using OperatorHub, the web console auto-populates the suggested namespace for the cluster administrator during the installation process.
```

Comment:
". . . in the YAML file" ? with an article + noun? 

auto-populates is probably OK. It's used 2 times in the current docs. "automatically populates" might be worth consideration, which is used ~8 times in the docs.